CONTRIBUTING.md

```diff
@@ -1,118 +1,182 @@
+
+Finally, you might want to have an Integrated Development Environment (IDE)
+that makes editing the source files easier.
+If you are hesitant on what IDE or editor to use,
+have a look at PyCharm, Visual Studio Code, or Spyder.
+
+### Work from a cloud environment
+
+There are some cloud options that give you
+the flexibility of a powerful IDE with a terminal,
+all from your web browser so you don't have to install anything.
+Two popular cloud environments for development are
+[the GitHub web editor](https://github.dev/poliastro/poliastro)
+and [Gitpod](https://gitpod.io/#https://github.com/poliastro/poliastro/).
+
+## Command line instructions
+
+In the sections below you can read step-by-step guides to
+perform common development tasks on the command line.
+All these instructions are meant for UNIX-like operating systems,
+for example:
+
+- Linux (any distribution will work)
+- macOS
+- Windows Subsystem for Linux (WSL)
+
+### Set up a development environment
+
+To set up a development environment you will need git and Python
+up and running. You should only need to do this once!
+
+Start by setting up git:
 
 1. [Install git](https://git-scm.com/) on your computer.
-2. [Register to GitHub](https://github.com/).
+2. [Sign up to GitHub](https://github.com/).
 3. [Fork poliastro](https://help.github.com/articles/fork-a-repo/).
-4. [Clone your fork](https://help.github.com/articles/cloning-a-repository/).
-5. Install `flit` using `pip install flit`.
-6. Install it in development mode using `flit install`, or
-   alternatively `flit install --symlink` (this means that
-   the installed code will change as soon as you change it in the
-   download location).
-7. Run `tox -e reformat` to make the format consistent.
-8. Run `tox -e check` to check all the formatting is right.
-   (The reformat command will not deal with un-used imports)
-9. Create a new branch.
-10. Make changes and commit.
-11. [Push to your
-    fork](https://help.github.com/articles/pushing-to-a-remote/).
-12. [Open a pull
-    request!](https://help.github.com/articles/creating-a-pull-request/)
-
-For more detailed explanations, please check out the [Astropy
-development
-docs](http://docs.astropy.org/en/stable/development/workflow/development_workflow.html).
+4. [Clone your fork](https://help.github.com/articles/cloning-a-repository/)
+   (remote name will be `origin`)
+5. Add an `upstream` remote with `git remote add upstream https://github.com/poliastro/poliastro.git`
+   and fetch its information with `git fetch upstream`
+6. Set your `main` branch to track `upstream` using `git branch --set-upstream-to=upstream/main`
+
+Next, configure your Python environment:
+
+6. Install Python for development.
+7. Create a Python virtual environment using `python -m venv .venv`
+8. Activate it using `source .venv/bin/activate`
+9. Upgrade the development dependencies using `python pip install -U pip setuptools wheel flit tox`
```

Comment:
Shouldn't this line be `python -m pip...`? 🤔

Reply:
In a non-Windows environment it's not that important, and also the virtual environment is activated at this point, but you're right that the core devs are trying to push for it more, and it's only a bit of extra typing. Will change it.

Reply:
Hah, I see what you meant - I forgot the `-m` 🙈